### ISSUE TITLE
"Data must be loaded before reading" error while using next() on RowIterator.php

### ISSUE THREAD

**Issue by Huub-Exel-Vonq** [openspout/openspout#271]
Heyy,

I am using the RowIterator from the XLSX directory.
I want to skip the first row.
So I thought I'd use ->next()
But for some reason when I use ->next() I get the "Data must be loaded before reading" error.
The reader is still open.

So this is a short summary of the code.
```php
$reader = ReaderEntityFactory::createXLSXReader(); // This does not give null, it gives the reader
$reader->open($filePath);                                              // No mistakes here either

foreach ($reader->getSheetIterator() as $sheet) {
    if ($sheet->getName() === $sheetName) {
        $sheetToUse = $sheet                                           // This also finds the right sheet!
    }
}

$sheetToUse->next();
```

And this fails and delivers the error: "Data must be loaded before reading"

Does anyone know why this happens, is the RowIterator not set correctly when you construct it?

Kind regards, Huub

**Comment by alexeitaber25** [comment]
$reader->getSheetIterator()->rewind(); helped me

**Comment by Huub-Exel-Vonq** [comment]
@alexeitaber25 
Yes! I also found this, but my question is why do I have to do this?
Why doesn't the sheetIterator start at 0 when getSheetIterator gets called? (right now it is set to nothing, or not set at all, when getSheetIterator is called). 

There is no functionality to tell it to, for example, start at 3.
Not setting the sheetIterator when you get it, does not have an advantage.

So doing this does not make any sense, is what I am trying to say.
